calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+| --------------------------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `start_time`                      | date              | Start time of log collection in UNIX timestamp format.                                                                                                                                                                                                                                                                                               |
+| `end_time`                        | date              | End time of log collection in UNIX timestamp format.                                                                                                                                                                                                                                                                                                 |
+| `action`                          | keyword           | - `allow`: Calico accepted the flow.<br />- `deny`: Calico denied the flow.                                                                                                                                                                                                                                                              |
+| `bytes_in`                        | long              | Number of incoming bytes since the last export.                                                                                                                                                                                                                                                                                                      |
+| `bytes_out`                       | long              | Number of outgoing bytes since the last export.                                                                                                                                                                                                                                                                                                      |
+| `dest_name`                       | keyword           | Contains one of the following values:<br />- Name of the destination pod.<br />- Name of the pod that was aggregated or the endpoint is not a pod. Check <code>dest_name_aggr</code> for more information, such as the name of the pod if it was aggregated.                                                                                         |
```

Comment:
There's no `dest_name_aggr` in OSS Flows - `dest_name` is effectively `dest_name_aggr` semantically, though. 

Note that throughout here, we use "Pod", but in actuality this means a collection of pods that share a name prefix (e.g., a Deployment).

For a concrete example, `dest_name` might be `my-service-abcd-*` and the pods within that deployment would have names like `my-service-abcd-1` and `my-service-abcd-2`

Reply:
Thanks for the clarification. I removed the `dest_name_aggr` line.

I think for now I'll stick with the current 'pod' usage. But I'll make a note to clarify this here and in the other flow logs documentation in CE and CC.